from-env-derive/tests/macro.rs

```diff
@@ -0,0 +1,131 @@
+use init4_from_env_derive::FromEnv;
+
+#[derive(FromEnv, Debug)]
+pub struct FromEnvTest {
```

Comment:
we should probably add a test with a tuple struct.

Reply:
done :)